infrastructure/prx_models/models/aruco_mip_36h12/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+
+
+execute_process(
```

Comment:
Would it be possible to check if the aruco XMLs are already present before executing this process?